<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Двухпросмотровый ассемблер в абсолютном формате (Горшков Ан.О., гр. 4412, 2024) </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(208, 208, 208);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </widget>
   <widget class="QPushButton" name="first_pass_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>163</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #E0E0E0; 
    border-radius: 5px; 
    border: 2px solid;
    border-color: white #8a8a8a #8a8a8a white;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #CCCCCC;  /* Цвет фона при наведении */
}

QPushButton:pressed {
    background-color: #B0B0B0;
    border-color: #A0A0A0 white white #A0A0A0;
}
</string>
    </property>
    <property name="text">
     <string>Первый проход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="second_pass_pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>750</y>
      <width>153</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #E0E0E0;
    border-radius: 5px; 
    border: 2px solid;
    border-color: white #8a8a8a #8a8a8a white;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #CCCCCC;  /* Цвет фона при наведении */
}

QPushButton:pressed {
    background-color: #B0B0B0;
    border-color: #A0A0A0 white white #A0A0A0;
}
</string>
    </property>
    <property name="text">
     <string>Второй проход</string>
    </property>
   </widget>
   <widget class="QLabel" name="choice_example_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>730</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбор примера</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="choice_example_comboBox">
    <property name="geometry">
     <rect>
      <x>591</x>
      <y>760</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>282</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #ffffff;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
    padding: 4px 25px 4px 8px;
}

QComboBox::drop-down {
    background-color: rgb(208, 208, 208);
    border: 2px solid #8a8a8a;
    border-top: 2px solid #f5f5f5;
    border-left: 2px solid #f5f5f5;
    border-bottom: 2px solid #8a8a8a;
    border-right: 2px solid #8a8a8a;
    width: 20px;
    height: 20px;
    border-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/resources/img/drop_arrow.png);
    width: 12px;
    height: 12px;
}

QComboBox::drop-down:hover {
    background-color: #b4b4b4;
}

QComboBox::drop-down:pressed {
    background-color: #a5a5a5;
}

QComboBox::drop-down:open {
    background-color: #a5a5a5;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    selection-background-color: #b4b4b4;
    selection-color: black;
    border: 1px solid #a5a5a5;
    outline: none;
    border-radius: 4px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #a5a5a5;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #c8c8c8;
    color: black;
}

</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>18</y>
      <width>321</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    border-top: 2px solid white;
    border-left: 2px solid white;
    border-right: 2px solid darkgray;
    border-bottom: 2px solid darkgray;
}
</string>
    </property>
    <widget class="QLabel" name="table_operation_codes_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>251</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Таблица кодов операций</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_operation_codes_tableWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>400</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
}

QTableWidget::item:selected {
    background-color: rgb(0, 0, 113);
	border: none;
	border-radius: 0px;
}

</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>95</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>95</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QTextEdit" name="source_text_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
    background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
}</string>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::NoWrap</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="source_text_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Исходный текст</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>18</y>
      <width>321</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    border-top: 2px solid white;
    border-left: 2px solid white;
    border-right: 2px solid darkgray;
    border-bottom: 2px solid darkgray;
}
</string>
    </property>
    <widget class="QLabel" name="auxiliary_table_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Вспомогательная таблица</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="auxiliary_table_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>291</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>200</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
}

QTableWidget::item:selected {
    background-color: rgb(0, 0, 113);
}
</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="table_symbolic_names_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Таблица символических имен</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_symbolic_names_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>301</width>
       <height>191</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
}

QTableWidget::item:selected {
    background-color: rgb(0, 0, 113);
}
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>125</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>125</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>26</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>26</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QTextBrowser" name="first_pass_errors_textBrowser">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>301</width>
       <height>93</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>9</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
	color: red;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="first_pass_errors_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>281</width>
       <height>19</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Ошибки первого прохода</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>18</y>
      <width>321</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    border-top: 2px solid white;
    border-left: 2px solid white;
    border-right: 2px solid darkgray;
    border-bottom: 2px solid darkgray;
}
</string>
    </property>
    <widget class="QTextBrowser" name="second_pass_errors_textBrowser_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>299</width>
       <height>93</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
    background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
	color: red;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="second_pass_errors_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>570</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Ошибки второго прохода</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="binary_code_textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>281</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
    background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
}</string>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::NoWrap</enum>
     </property>
    </widget>
    <widget class="QLabel" name="binary_code_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>181</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Двоичный код</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="object_module_header_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>299</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: white;
    border-top: 2px solid #8a8a8a;
    border-left: 2px solid #8a8a8a;
    border-right: 2px solid #e0e0e0;
    border-bottom: 2px solid #e0e0e0;
    border-radius: 5px;
}

QTableWidget::item:selected {
    background-color: rgb(0, 0, 113);
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>95</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>95</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>35</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>35</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QLabel" name="object_module_header_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>273</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: none;
}</string>
     </property>
     <property name="text">
      <string>Заголовок объектного модуля</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
